packages/components-js/src/pages/scroller.html

```diff
@@ -193,3 +193,30 @@
     </p-scroller>
   </div>
 </div>
+
+<div class="playground light" title="should render scroller with scrollbar true">
+  <p-scroller scrollbar="true">
+    <p-text-list>
```

Comment:
Please use native elements with custom styles so that changes to `p-text-list` don't affect the unrelated scroller tests.

Reply:
Fixed it